src/main/sensors/compass.c

```diff
@@ -258,6 +242,22 @@ bool compassDetect(magDev_t *dev)
 #endif
         FALLTHROUGH;
 
+    case MAG_QMC5883:
+#ifdef USE_MAG_QMC5883
+        if (busdev->bustype == BUSTYPE_I2C) {
+                busdev->busdev_u.i2c.address = compassConfig()->mag_i2c_address;
```

Comment:
Unnecessary double indentation.

Reply:
I was only moving already existing code around. Anyhow have fixed this.